lib/commands/CreateLauncherPR.ts

```diff
@@ -7,89 +7,109 @@ import {
   MappedParameter,
   MappedParameters,
   Parameter,
+  Project,
+  RepoRef,
   Secret,
   Secrets,
+  SimpleProjectEditor,
   success,
 } from "@atomist/automation-client";
-import {CommandHandler} from "@atomist/automation-client/lib/decorators";
-import {HandleCommand} from "@atomist/automation-client/lib/HandleCommand";
-import {relevantRepos} from "@atomist/automation-client/lib/operations/common/repoUtils";
-import {editOne} from "@atomist/automation-client/lib/operations/edit/editAll";
-import {BranchCommit} from "@atomist/automation-client/lib/operations/edit/editModes";
-import {allReposInTeam} from "@atomist/sdm";
-import {DefaultRepoRefResolver} from "@atomist/sdm-core";
-import {BOOSTER_CATALOG_REPO, SNOWDROP_ORG} from "../constants";
-import {updateLauncherCatalog} from "../support/catalog/updateLauncherCatalog";
-import {DefaultLatestTagRetriever} from "../support/github/boosterUtils";
```

Comment:
Does removing the `LatestTagRetriever` has to do with the automation client calculating wrong the latest version? #73

Reply:
Yes. I've extracted the logic to a separate place and started using an interface for abstraction https://github.com/snowdrop/snowdrop-automation-client/pull/85/files#diff-cb9f91a06d665c423e70ab3c87133645R1.
Also, `LatestTagRetriever` used to get the tags from github + filter them. I've split this logic and now tag finder only filters an array that is given to it. This way it's easier to test it.